.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 Please contribute feedback on the overall design of WebAssembly, issues about
 the textual specification, or typos from the docs section of webassembly.org.
 
-For browser issues or tooling bugs, see links at http://webassembly.org/community/feedback/.
\ No newline at end of file
+For browser issues or tooling bugs, see links at https://github.com/WebAssembly/website/blob/master/community/feedback.md.
```

Comment:
@jfbastien I do not believe the CG has 'consensus that was previously reached', there are many important matters of business for the CG in the roadmap and I did not see a heads up on these and thus I believe linking to these would misrepresent the CG. Sorry I do not find your views on this matter believable and I note you have failed to respond to repeated requests to explain your claim to a position of Chair or to return this Chair to the CG and that moving to a W3C working group would take this decision away from the CG members. Obviously this is all heading to the CG mailing list for the CG members to sort out, but the members may not be kind if good faith attempts to do the right thing have not been made, and I am happy to give people more time.